dev-docs/modules/userId.md

```diff
@@ -74,44 +74,34 @@ of sub-objects. The table below has the options that are common across ID system
 
 ## User ID Sub-Modules
 
-### Criteo RTUS
+### Criteo ID
 
-Criteo Real Time User Sync (RTUS) is designed for use as an alternative for platforms that cannot drop their cookies due to Safari 3rd party restriction.
+Criteo ID module is designed for use as an alternative for platforms that cannot drop their cookies due to Safari 3rd party restriction.
 
 Add it to your Prebid.js package with:
 
 {: .alert.alert-info :}
-gulp build --modules=userId,criteortusIdSystem
+gulp build --modules=userId,criteoIdSystem
 
-#### Criteo RTUS Registration
+#### Criteo ID Configuration
 
-In order to use a Criteo rtus id a bidder must reach out to Criteo and get their unique client identifier.
+The Criteo ID module does not require any configuration parameters. It should work as-is provided that bidders forward it in their adapters.
 
-#### Criteo RTUS Configuration
-
-{: .table .table-bordered .table-striped }
-| Param under userSync.userIds[] |  Scope   |  Type  |  Description                                                        |  Example             |
-|--------------------------------|----------|--------|---------------------------------------------------------------------|----------------------|
-| params                         | Required | Object | Details of Criteo ID                                                |                      |
-| params.clientIdentifier        | Required | Object | Object containing bidder code as key and client identifier as value | `{ "appnexus": 30 }` |
+List of the working bidders:
+* TBD
 
 {: .alert.alert-info :}
-NOTE: Criteo user id's max age is 1 hour. Criteo rtus module makes a request to criteo endpoint every hour to fetch new user id. Do not use `params.storage` when adding configuration for criteortus. If you are using multiple id systems then you can use storage (if that id system supports it). Read more about the `storage` property under [Basic Configuration](#basic-configuration).
+NOTE: In order to be fully functional, the Criteo ID module should be called as often as possible. Do not use `params.storage` with this module as it could lower the performances. If you are using multiple id systems then you can use storage (if that id system supports it). Read more about the `storage` property under [Basic Configuration](#basic-configuration).
```

Comment:
It should interest the reader to know that while we suggest call the module as often as possible, it embeds its own caching mechanism. Here's a wording proposition:

For optimal performance, the Criteo Id module should be called at every opportunity. It embeds its own optimal caching mechanism. It's best not to use `params.storage` with this module as it may only lower the performances. If you are using multiple id systems, however, you may use it for the other id systems that supports it.